test/endToEnd/nodeSplitTest.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const loginService = require('./util/loginService');
+const analysesService = require('./analyses/analysesService');
+const modelService = require('./models/modelService.js');
+
+const TITLE = 'my title';
+const OUTCOME = 'my outcome';
+const NODE_SPLIT_MODEL_TITLE = 'Nodesplit model (Fluoxetine - Paroxetine)';
+
+function beforeEach(browser) {
+  browser.resizeWindow(1366, 728);
+  loginService.login(browser)
+    .waitForElementVisible('#analyses-header');
+  analysesService.addAnalysis(browser, TITLE, OUTCOME, '/example.json');
+  modelService.addDefaultModel(browser)
+    .click('#node-split-overview-link')
+    .waitForElementVisible('#nodesplit-overview-header');
+  browser
+    .assert.containsText('#analysis-title-header', TITLE)
+    .assert.containsText('#outcome-header', OUTCOME)
+    .assert.containsText('#effects-type', 'random')
+    .assert.containsText('#imputed-outcome-scale', '1.1321 (imputed)')
+    .assert.containsText('#likelihood-link', 'binom / logit')
+    .assert.containsText('#consistency-model-title', 'title')
+    .assert.containsText('#random-effects-standard-deviation', '0.3').pause(100)
+    .assert.containsText('#deviance-information-criterion', '26.');
+
+  browser.click('#create-node-split-model-button-0')
+    .waitForElementVisible('#create-model-header');
+}
+
+function afterEach(browser) {
+  analysesService
+    .deleteFromList(browser)
+    .end();
+}
+
+function createNodeSplit(browser) {
```

Comment:
This is missing a test for creating the consistency model based on a node-split (that order also works)

so 1) create node-split
2) click NS overview
3) click 'create consistency model' button

Reply:
that functionality seems to be broken atm